tests/integration/test_requests.py

```diff
@@ -197,12 +191,12 @@ def test_nested_cassettes_with_session_created_before_nesting(scheme, tmpdir):
         assert session.get(url).content == third_body
 
     # Make sure that the session can now get content normally.
-    session.get('http://www.reddit.com')
+    session.get(httpbin_both.url)
```

Comment:
maybe..?  to closely match the prior intention, request something else, like `httpbin_both + '/robots.txt'` and assert it contains `User-agent: *`